### DISCUSSION TITLE
Combat Logging

### DISCUSSION THREAD

**Discussion by DaltonCrawley** [Xzandro/sw-exporter#505] [Ideas]
Hi, I am learning programming so I am sticking to Python for now. I was wondering if there was a way to record moves and their effects during combat. I know that a feature currently exists that logs the runtimes and success of dungeon runs/failures; However, a more specific log about the moves that happen during battle would be a very insightful feature.

**Comment by Xzandro** [comment]
There isnt at least not with SWEX, everything combat-related (except for RTA) is caluclated on the client side only, so nothing is sent to the server.

**Reply by DaltonCrawley** [reply]
Do you have any ideas as to how I would try to log the combat-related things on the client side?

**Reply by Xzandro** [reply]
Not really, probably some reverse-engineering / memory stuff. Certainly not my field of expertise. I'm also not really too interested in it.
